test/plugin/scenarios/resttemplate-4.x-scenario/support-version.list

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+1.5.0.RELEASE
```

Comment:
I am confused, the plugin called restteamplate 4.x, why all these versions are `1.5.x`? From test/plugin/scenarios/resttemplate-4.x-scenario/pom.xml, you are switching the spring boot version.

Could you explain why?

Reply:
From the plugin pom codes, clearly, it is spring version of 4.x

![image](https://user-images.githubusercontent.com/5441976/76164941-e9433400-618d-11ea-8749-239dae87a195.png)

Reply:
`<dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-web</artifactId>
            <version>${test.framework.version}</version>
        </dependency>`
In resttemplate scenario, use this denpendency.

Reply:
Yes, I read that, and that is my question. Why we set these versions? We are testing different rest template versions, not spring versions. We should be able to use maven to set explicit rest template versions and use the same spring version at the same time.

Reply:
Please correct me if I am wrong.